website/docs/reference/widgets/list.md

```diff
@@ -212,23 +201,111 @@ These properties allow you to bind your List widget with any other widget in que
 | Binding Property    | Description                                                                                                                                                                                                                               |
 | ------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | **backgroundColor** | Represents the widget's **Background Color** setting as a CSS `color` value _(string)_.                                                                                                                                                   |
-| **gridGap**         | Reflects the widget's **Item Spacing** property _(number)_.                                                                                                                                                                               |
+| **itemSpacing**         | Reflects the widget's ¸ property _(number)_.                                                                                                                                                                               |
 | **isVisible**       | Reflects the state of the widget's **Visible** setting _(bool)_.                                                                                                                                                                          |
-| **items**           | Contains an *array* of *objects* that each represent a widget within the list items, and holds information about that widget's state.<br/>e.g. `[ { "Text1": { "isVisible": true, ... }, ... }, ... ]` |
+| **currentItemsView**           | Contains an *array* of *objects* that each represent a widget within the list items, and holds information about that widget's state. <br/>e.g. `[ { "Text1": { "isVisible": true, ... }, ... }, ... ]` <br/>  |
```

Comment:
Is this an internal property, Maybe we should list in down under `Internal Binding Properties` cc @ashit-rath

Reply:
This is not an internal property but a List's property